### DISCUSSION TITLE
Quad9: DoH

### DISCUSSION THREAD

**Discussion by lokamaya** [bebasid/KominFudge#76] [Ideas]
Salam. Terima kasih teman-teman yang sudah meluangkan waktu dan tenaga untuk bebasID.

Saya sudah komunikasi dengan support system Quad9 dan menyampaikan kecurigaan bahwa "Quad9 dibelokin di level IP Transit", lebih tepatnya di IIX. Jawaban Quad9:

> These IPs are our servers in Jakarta, as AS42 (WoodyNet, PCH, Packet Clearing House, etc,) is our upstream, and are present at the IIX Jakarta Internet Exchange (see https://docs.quad9.net/FAQs/#network-providers-dns-leak-tests).

Jadi Quad9 sepertinya aman dipakai.

Semoga membantu.
Cheers.

**_Traceroute:_**
![Capture1](https://github.com/user-attachments/assets/d5c1c47e-f425-47b0-af06-e50f2671b3fd)

**_IPInfo_**
![Capture3](https://github.com/user-attachments/assets/67bf86ab-89f9-4064-a72b-1c6a423249d1)

**Comment by ryukora** [comment]
> Salam. Terima kasih teman-teman yang sudah meluangkan waktu dan tenaga untuk bebasID.
> 
> Saya sudah komunikasi dengan support system Quad9 dan menyampaikan kecurigaan bahwa "Quad9 dibelokin di level IP Transit", lebih tepatnya di IIX. Jawaban Quad9:
> 
> > These IPs are our servers in Jakarta, as AS42 (WoodyNet, PCH, Packet Clearing House, etc,) is our upstream, and are present at the IIX Jakarta Internet Exchange (see https://docs.quad9.net/FAQs/#network-providers-dns-leak-tests).
> 
> Jadi Quad9 sepertinya aman dipakai.
> 
> Semoga membantu. Cheers.
> 
> **_Traceroute:_** ![Capture1](https://private-user-images.githubusercontent.com/39824/380350094-d5c1c47e-f425-47b0-af06-e50f2671b3fd.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyMDExODIsIm5iZiI6MTczMDIwMDg4MiwicGF0aCI6Ii8zOTgyNC8zODAzNTAwOTQtZDVjMWM0N2UtZjQyNS00N2IwLWFmMDYtZTUwZjI2NzFiM2ZkLkpQRz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI5VDExMjEyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM2ZjhiMWI2NjQzODJmMWY2ODBiOGM2NTUyNWFlMzgzMjQ1NDcxMGZjZDBhZGIzMzViMzNlYWI1NDdkOTk4YzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.PAjGlAJ5rPvvaMa0U1b7h2z8rNOz_8OwKILIxMPxfeY)
> 
> **_IPInfo_** ![Capture3](https://private-user-images.githubusercontent.com/39824/380350188-67bf86ab-89f9-4064-a72b-1c6a423249d1.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyMDExODIsIm5iZiI6MTczMDIwMDg4MiwicGF0aCI6Ii8zOTgyNC8zODAzNTAxODgtNjdiZjg2YWItODlmOS00MDY0LWE3MmItMWM2YTQyMzI0OWQxLkpQRz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI5VDExMjEyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVmNzhmYzliNjE5ZGQ1ZmNkNTQwYWM5ZTM4NzBlNTgzMGE1YWZjMmNkZjQ0ZDJhMTE3ZjNjMTU4OWI3NzBhNzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.bgntXInBtugt-Vgz_XkopJuV4nkmXqV4dhe8lldPabI)

CC: @merdekaid
